index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="title">
+      <a href="/"><img src="./image/panda_market.png" alt="판다마켓 로고" class="logo"></a>
+      <a href="./sign-in.html"><button class="login-button">로그인</button></a> 
+    </div>
+ 
+<div class="container0">
+    <!-- 텍스트와 사진 -->
+  <div class="inner_box0"> 
+    <div class text-box0>
```

Comment:
❗️ 수정요청
오타인 것 같아요~ 확인 후 수정하세요!

```suggestion
    <div class="text-box0">
```